matRad_dijSampler.m

```diff
@@ -0,0 +1,63 @@
+function [cst,dij] = matRad_dijSampler(cst,ct,dij, margin_size,sparCity)
```

Comment:
usually we use letter case-separated words e.g. marginSize for variable names in matRad...